main/conceptual/getting-started/start-a-project.md

```diff
@@ -25,18 +21,18 @@ We'll be running **three terminal windows**. See below:
 ::: tip Watch: Prepare Your Agoric Environment (November 2020)
 This presentation includes starting a project, but note an outdated detail:
 
- - In the REPL `x~.go()` tildot support has been postponed; use `E(x).go()`.
+ - In the REPL `x~.go()` tilde support has been postponed; use `E(x).go()` instead.
```

Comment:
"tildot" means `~.`; if you prefer not to use / introduce that term, the replacement isn't just "tilde" but something like "tilde-dot".

Reply:
! I've never seen the word "tildot" before; I thought it was a typo. I reverted the text back to "tildot"

Reply:
I think we made it up as part of the discussion of this JavaScript syntax extension.